example_app_generator/run_specs.rb

```diff
@@ -7,7 +7,6 @@
 run('bin/rake --backtrace spec:requests') || abort
 run('bin/rake --backtrace spec:models') || abort
 run('bin/rake --backtrace spec:views') || abort
-run('bin/rake --backtrace spec:controllers') || abort
```

Comment:
Could you explain why we have to change that?

Reply:
I suspected that the smoke app does not generate controller specs anymore because the default of the ScaffoldGenerator changed

Reply:
Rather than removing these, you need to add a command to generate some controller tests for this to run.

Reply:
Done.